src/index.js

```diff
@@ -91,22 +90,32 @@ const puppeteerConfig = {
  * ==============================================================
  */
 async function main () {
-  console.log(colors.magenta.bold('Launching ReLaXed...'))
-  const browser = await puppeteer.launch(puppeteerConfig);
-  const page = await browser.newPage()
+    console.log(colors.magenta.bold('Launching ReLaXed...'))
+    const browser = await puppeteer.launch(puppeteerConfig);
+    const page = await browser.newPage()
 
-  page.on('pageerror', function (err) {
-    console.log(colors.red('Page error: ' + err.toString()))
+    if(fs.existsSync(path.join(inputDir, '.relaxed.json'))) {
+        await plugin.loadPlugins(path.join(inputDir, '.relaxed.json'))
```

Comment:
The "." at the beginning could be confusing for non-developers, I would go for something more straightworfard: ``config.yaml``or ``settings.yaml``